src/UI/Implementation/Component/Dropdown/Renderer.php

```diff
@@ -10,6 +10,12 @@
 use ILIAS\UI\Implementation\Render\ResourceRegistry;
 
 class Renderer extends AbstractComponentRenderer {
+
+	/**
+	 * @var string
+	 */
+	private $class_name = 'dropdown';
```

Comment:
Please have a look into the description of this PR and why we introduced this pattern. We want to use the code of the dropdown, but still be able to identify it as being a dropdown for sortation. What would be your pick to achieve that.